raiden/raiden_service.py

```diff
@@ -391,28 +845,33 @@ def open(
 
         if settle_timeout < self.raiden.config['settle_timeout']:
             raise ValueError('Configured minimum `settle_timeout` is {} blocks.'.format(
-                self.raiden.config['settle_timeout']))
+                self.raiden.config['settle_timeout']
+            ))
+
+        token_address_bin = token_address.decode('hex')
```

Comment:
`_bin` vs `_hex` as above. but on second sight: it seems we haven't established the `_bin` sandwich boundary at `RaidenAPI` as we planned yet. that's not part of your PR then.